Products/CMFPlone/controlpanel/browser/redirects.py

```diff
@@ -163,7 +163,7 @@ def view_url(self):
 
 
 class RedirectionSet:
-    def __init__(self, query="", created="", manual=""):
```

Comment:
For backwards compatibility reasons, we should still accept the `created` parameter, log a deprecation warning, and use it as the `start` parameter if it is not empty.

In the tests, we should try to keep the existing tests, and add new ones for the new parameters. This helps confirm that we haven't broken backwards compatibility.